domains/transfer/Transfer.API/API/ClaimAutomation/Api/Controllers/ClaimAutomationController.cs

```diff
@@ -14,21 +15,23 @@
 
 namespace API.ClaimAutomation.Api.Controllers;
 
-[Authorize]
 [ApiController]
-[ApiVersion(ApiVersions.Version20240103)]
+[Authorize(Policy.Frontend)]
+[ApiVersion(ApiVersions.Version20240515)]
 [Route("api/claim-automation")]
-public class ClaimAutomationController(IUnitOfWork UnitOfWork) : ControllerBase
+public class ClaimAutomationController(IUnitOfWork unitOfWork, AccessDescriptor accessDescriptor) : ControllerBase
 {
-    [Authorize(Policy = PolicyName.RequiresCompany)]
     [HttpPost("start")]
     [ProducesResponseType(typeof(ClaimAutomationArgumentDto), 201)]
     [ProducesResponseType(typeof(ClaimAutomationArgumentDto), 200)]
-    public async Task<ActionResult> StartClaimAutomation()
+    public async Task<ActionResult> StartClaimAutomation([FromQuery] Guid organizationId)
     {
-        var user = new UserDescriptor(User);
+        if (!accessDescriptor.IsAuthorizedToOrganization(organizationId))
```

Comment:
Just general question. Was if (!accessDescriptor.IsAuthorizedToOrganization(organizationId)) the thing we wanted all places that uses organization id? Can't really remember where we ended on this. @martinhenningjensen @sahma19 .

Reply:
I can't remember.